File: app/src/main/java/wxm/dofcalculator/ui/calculator/event/ObjectDistanceRangeChangeEvent.java
package wxm.dofcalculator.ui.calculator.event;

/**
 * 物距范围调整事件
 * Created by ookoo on 2017/3/22.
 */
public class ObjectDistanceRangeChangeEvent {
    private int         mODMin;
    private int         mODMax;

    public ObjectDistanceRangeChangeEvent(int min, int max)   {
        mODMin = min;
        mODMax = max;
    }

    /**
     * 获取最小物距(单位 : 米)
     * @return     最小物距
     */
    public int getObjectDistanceMin()   {
        return mODMin;
    }

    /**
     * 获取最大物距(单位 : 米)
     * @return      最大物距
     */
    public int getObjectDistanceMax()   {
        return mODMax;
    }
}

File: app/src/main/java/wxm/dofcalculator/ui/calculator/event/ObjectDistanceRangeChangeEvent.kt
package wxm.dofcalculator.ui.calculator.event

/**
 * for object distance
 * [objectDistanceMin] min distance, unit is m
 * [objectDistanceMax] max distance, unit is m
 * Created by WangXM on2017/3/22.
 */
class ObjectDistanceRangeChangeEvent(
        val objectDistanceMin: Int,
        val objectDistanceMax: Int)
